index.html

```diff
@@ -158,27 +178,28 @@ <h3 class="card-main-title">Profesional Art Printing Data</h3>
                                <li><button type="button" class="btn-card">Bootstrap</button></li>
                                <li><button type="button" class="btn-card">Ruby</button></li>
                             </ul>
-                        </section>
-                            
+                        </section>  
                         </section> 
                         <button type="button" class="card-big-btn" >See project</button>    
                     </section>
                     
             </section> 
         </article>  
         <article id="work-experience">
-            <section class="background-image">
             <section class="About-me">
+               <section class="About-content ">
                <h3 class="about-title">About me</h3>
                <p class="about-description">Hello I’m a software developer! I can help you build a product , 
                feature or website Look through some of my work and experience!
                If you like what you see and have a project you need coded, 
                don’t hesitate to contact me.
                </p>
                <button type="button" class="get-my-resume">Get my resume</button>
-               <section class="line"></section> 
-            </section> 
+            </section>
+            
         </section>
+        <div class="line"><hr></section> 
+               
             <section class="work-projects">
                 <section>
                     <section class="figures"><section class="diamond"></section><span class="work-project-title">Language</span></section>
```

Comment:
- [ ]  You are doing a great job so far but I think you can still improve this section better to closely look like the Figma design. 💯
- - On screen sizes between `764px` and `1130px` the content of your `work-projects` overflows the width of the screen, please consider fixing this as good and responsive UI is paramount to the overall user experience of your website. 😄 👍
Screenshot:
![Screenshot 2023-04-01 094609](https://user-images.githubusercontent.com/107632780/229277453-8ab53c5e-9194-4e20-91e1-3b7deb093230.png)